src/commonMain/kotlin/fr/acinq/lightning/channel/states/Normal.kt

```diff
@@ -380,8 +484,8 @@ data class Normal(
                                 val nextState = this@Normal.copy(spliceStatus = SpliceStatus.InProgress(replyTo = null, session, localPushAmount = 0.msat, remotePushAmount = cmd.message.pushAmount, origins = cmd.message.origins))
                                 Pair(nextState, listOf(ChannelAction.Message.Send(spliceAck)))
                             } else {
-                                logger.info { "rejecting splice attempt: channel is not idle" }
-                                Pair(this@Normal.copy(spliceStatus = SpliceStatus.Aborted), listOf(ChannelAction.Message.Send(TxAbort(channelId, InvalidSpliceChannelNotIdle(channelId).message))))
+                                logger.warning { "cannot initiate splice, channel is not quiescent" }
```

Comment:
```suggestion
                                logger.warning { "rejecting splice attempt: channel is not quiescent" }
```

Reply:
fixed in [982d5d2](https://github.com/ACINQ/lightning-kmp/pull/558/commits/982d5d22fa484ef7be1bb7925664d50f32a4ca4b)